src/lib/ukraine-districts/districts-list.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  сrimea,
+  mykolayiv,
+  chernihiv,
+  rivne,
+  chernivtsi,
+  ivanoFrankivsk,
+  khmelnytskyi,
+  lviv,
+  ternopil,
+  uzhhorod,
+  lutsk,
+  cherkassy,
+  kirovograd,
+  kyiv,
+  odessa,
+  vinnytsya,
+  zhytomyr,
+  sumy,
+  dnipropetrovsk,
+  donetsk,
+  kharkiv,
+  luhansk,
+  poltava,
+  zaporizhzhya,
+  kyivCity,
+  kherson,
+  sevastopol,
+} from "./districts-path-data";
+
+export const distractsList = [
+  сrimea,
+  mykolayiv,
+  chernihiv,
+  rivne,
+  chernivtsi,
+  ivanoFrankivsk,
+  khmelnytskyi,
+  lviv,
+  ternopil,
+  uzhhorod,
+  lutsk,
+  cherkassy,
+  kirovograd,
+  kyiv,
+  odessa,
+  vinnytsya,
+  zhytomyr,
+  sumy,
+  dnipropetrovsk,
+  donetsk,
+  kharkiv,
+  luhansk,
+  poltava,
+  zaporizhzhya,
+  kyivCity,
+  kherson,
+  sevastopol,
+];
+
+export const showBottom = [
+  chernihiv.id,
+  sumy.id,
+  zhytomyr.id,
+  rivne.id,
+  lutsk.id,
+];
+
+export const showLeft = [donetsk.id, luhansk.id];
```

Comment:
Why do we need these lists?

Reply:
When you click on these areas, the popover goes beyond the black background and the white colors merge.